tests/pyrog/factories.py

```diff
@@ -30,6 +30,7 @@ class Meta:
     id = factory.Sequence(lambda n: f"resource_id_{n:04d}")
     source = factory.SubFactory(SourceFactory)
     primary_key_owner = factory.SubFactory("tests.pyrog.factories.OwnerFactory")
+    definition = factory.Faker("json")
```

Comment:
don't you need a bit more realistic value than just a random json ?

Reply:
don't know if we need a proper structure def for now